pkg/timeutils/utils.go

```diff
@@ -9,12 +9,8 @@ import (
 // or Unix timestamp, if successful returns a Unix timestamp
 // as string otherwise returns value back.
 func GetTimestamp(value string) string {
-	format := RFC3339NanoFixed
 	loc := time.FixedZone(time.Now().Zone())
-	if len(value) < len(format) {
```

Comment:
Why is code no longer needed?  I'm guessing it might have been there to remove any +07:00 at the end, but I'm not sure why since that's valid input per RFC3339